src/sqlancer/stonedb/gen/StoneDBTableCreateGenerator.java

```diff
@@ -51,11 +55,29 @@ public SQLQueryAdapter getQuery() {
             appendColumns();
             sb.append(" ");
             appendTableOptions();
-            addCommonErrors(errors);
+            addExpectedErrors();
             return new SQLQueryAdapter(sb.toString(), errors, true);
         }
     }
 
+    private void addExpectedErrors() {
+        // java.sql.SQLSyntaxErrorException: BLOB/TEXT column 'c0' used in key specification without a key length
+        errors.add("used in key specification without a key length");
+        // java.sql.SQLException: Tianmu engine does not support unique index.
+        errors.add("Tianmu engine does not support unique index");
+        // java.sql.SQLException: BLOB column 'c0' can't be used in key specification with the used table type
+        errors.add("can't be used in key specification with the used table type");
+        // java.sql.SQLException: Table storage engine for 't0' doesn't have this option
+        errors.add("doesn't have this option");
+        // java.sql.SQLSyntaxErrorException: Table 't0' uses an extension that doesn't exist in this MySQL version
```

Comment:
Why are we getting this error? Is this due to a feature that StoneDB does not support, but MySQL does? If there is no way to exercise the feature, we should probably not test it.

Reply:
@ZhengLin-Li, any updates on this?